### DISCUSSION TITLE
Distortion correction with field maps (rather than reverse phase images)

### DISCUSSION THREAD

**Discussion by dmccalley244** [muscbridge/PyDKE#261] [Technical Support]
Hi all, 

Does PyDesigner offer any way to correct for EPI distortions using field maps rather than reverse phase images? It seems like the --undistort won't work without reverse phase images collected 

Let me know what you all think! 

-Dan

**Comment by TheJaeger** [comment]
Hi Dan,

I apologize for responding so late. PyDesigner relies on MRtrix3's wrapper function _dwifslpreproc_ to undistort a DWI, and this wrapper currently does not accept field maps.

We're currently looking into incorporating synthetic reverse phase encoded B0s from [Synb0-DISCO](https://github.com/MASILab/Synb0-DISCO) into PyDesigner for undistortion. There's no ETA as of yet, but it's something we'll definitely include in later updates to PyDesigner. In the meantime, you may experiment with Synb0-DISCO to see if it may be apply to your imaging protocol.
